pom.xml

```diff
@@ -122,8 +122,8 @@ under the License.
 		<flink.forkCountUnitTest>4</flink.forkCountUnitTest>
 		<flink.reuseForks>true</flink.reuseForks>
 		<flink.surefire.baseArgLine>-XX:+UseG1GC -Xms256m</flink.surefire.baseArgLine>
-		<flink.shaded.version>16.1</flink.shaded.version>
-		<flink.shaded.jackson.version>2.13.4</flink.shaded.jackson.version>
```

Comment:
There's also `<curator.version/>` property further down in the file that is used for test code. I suggest that we keep these two dependencies in sync. We would have to change some parameter initialization (see [CURATOR-663](https://issues.apache.org/jira/browse/CURATOR-663)) if we decide to upgrade.

Reply:
> I suggest that we keep these two dependencies in sync.

Just to double check, so upgrade to Curator 4.0 as part of the upgrade to flink-shaded 17.0?
If so, can you help with the necessary changes?

Reply:
Ah yes, that's what I meant but it doesn't help if I type `4.0` instead of `5.4.0` 👍